consensus/quorum/one-node-staked-vote.go

```diff
@@ -0,0 +1,104 @@
+package quorum
+
+import (
+	"math/big"
+
+	"github.com/harmony-one/bls/ffi/go/bls"
+	"github.com/harmony-one/harmony/internal/common"
+	"github.com/harmony-one/harmony/numeric"
+	"github.com/harmony-one/harmony/staking/effective"
+)
+
+var (
+	twoThirds = numeric.NewDec(2).QuoInt64(3)
+)
+
+type stakedVoter struct {
+	isActive, isHarmonyNode bool
+	effective               numeric.Dec
+}
+
+type stakedVoteWeight struct {
+	SignatureReader
+	// EPOS based staking
+	validatorStakes            map[*bls.PublicKey]stakedVoter
+	totalEffectiveStakedAmount numeric.Dec
+}
+
+// Policy ..
+func (v *stakedVoteWeight) Policy() Policy {
+	return SuperMajorityStake
+}
+
+// We must maintain 2/3 quoroum, so whatever is 2/3 staked amount,
+// we divide that out & you
+// IsQuorumAchieved ..
+func (v *stakedVoteWeight) IsQuorumAchieved(p Phase) bool {
+	return true
```

Comment:
no TODO here?

Reply:
Didn't because will be implementing soon, should I do TODOs even for things I will be doing soon?